astropy/coordinates/baseframe.py

```diff
@@ -1150,3 +1341,68 @@ def __setattr__(self, name, value):
             raise AttributeError("can't set frame attribute '{0}'".format(name))
         else:
             super(GenericFrame, self).__setattr__(name, value)
+
+# Note: this is a base class that defines default representation info for frames
+# that represent longitude and latitude with RA, Dec
+class BaseRADecFrame(BaseCoordinateFrame):
+    """
+
+    Parameters
+    ----------
+    representation : `BaseRepresentation` or None
+        A representation object or ``None`` to have no data (or use the other
+        keywords below).
+
+    ra : `Angle`, optional, must be keyword
+        The RA for this object (``dec`` must also be given and ``representation``
+        must be None).
```

Comment:
This is actually not true: if the first argument is not a representation or `None`, the second will be taken to be `ra`...  Not sure to what extent we want to reflect reality here...